Online/State/PhysicalObjectEntityState.cs

```diff
@@ -65,6 +65,12 @@ public override void ReadTo(OnlineEntity onlineEntity)
             var apo = onlineObject.apo;
             RainMeadow.Trace($"{onlineEntity} received realized state? {realizedObjectState != null} entity realized?{onlineObject.realized}");
 
+            if (!realized && !onlineObject.realized && !onlineEntity.isTransferable)
```

Comment:
this assumes that if an entity is locked to one owner and it isn't realized in the owner's world then other players shouldn't be able to realize it

less hacky solution would be to have a `shouldRealize` bool then guard all the joinimpl/realize stuff

Reply:
This is a bit pointless because once an item is swallowed, the game stops updating it and moving it, so we'll receive a non-moving position anyways.

Reply:
without this check, clients will try to realize it in the room where it was originally swallowed

Reply:
I think they still will if the entity is already found there in abstractroom on room-load, regardless of the move() call here

Reply:
Realize will only be called if ChangeRooms is called, which will only be called if pos1.room != pos2.room

If we are feeding the same position after the removal, which we should, this here isn't what's causing it to realize.